3rdparty/SConscript

```diff
@@ -12,6 +12,7 @@ thirdparty_versions = {
     'libuv':            '1.35.0',
     'ltdl':             '2.4.6',
     'openfec':          '1.4.2.4',
+    'openssl':          '3.0.7', # minimal supported version is 1.1.1
```

Comment:
We maintain the list of minimal versions [here](https://roc-streaming.org/toolkit/docs/building/dependencies.html), let's remove this comment and instead add openssl to that page.

Reply:
done